content/docs/components/serving/kfserving.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "KFServing"
+description = "Using KFServing for serving Models"
+weight = 2
++++
+
+## KFServing with Kubeflow
+
+KFServing can be installed with Kubeflow v0.7, and KFServing Kustomize install files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
+
+KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and ONNX. 
+
+KFServing encapsulates the complexity of autoscaling, networking, health checking, and server configuration to bring cutting edge serving features like GPU Autoscaling, Scale to Zero, and Canary Rollouts to your ML deployments. It enables a simple, pluggable, and complete story for production ML Inference Server by providing prediction, pre-processing, post-processing and explainability out of the box.
+
+### Learn More
+* Join our [Working Group](https://groups.google.com/forum/#!forum/kfserving) for meeting invites and discussion.
+* [Read the Docs](https://github.com/kubeflow/kfserving/tree/master/docs).
+* [Examples](https://github.com/kubeflow/kfserving/tree/master/docs/samples).
+* [API Docs](https://github.com/kubeflow/kfserving/tree/master/docs/apis/README.md)
+* [Roadmap](https://github.com/kubeflow/kfserving/tree/master/ROADMAP.md).
+* [KFServing 101 Slides](https://drive.google.com/file/d/16oqz6dhY5BR0u74pi9mDThU97Np__AFb/view).
+
+### Prerequisits
+KNative Serving and Istio should be available on Kubernetes Cluster.
+- Istio Version: v1.1.7 + 
```

Comment:
This list is broken on the rendered page. Add an empty line before the first list item to fix.